packages/table/src/regions.ts

```diff
@@ -164,6 +165,13 @@ export class Regions {
     }
 
     /**
+     * Returns a region containing the entire table.
+     */
+    public static table(): IRegion  {
+        return {};
```

Comment:
This one: https://github.com/palantir/blueprint/blob/master/packages/table/src/regions.ts#L110